account_payment_credit_card/models/account.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2017 Open Source Integrators <http://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    credit_card = fields.Boolean('Transfer AP to Credit Card Company')
+    partner_id = fields.Many2one(
```

Comment:
This is quite generic field name, maybe we should consider refactoring this?

Reply:
@nikul-serpentcs any comments on `partner_id`? Maybe something like `card_processor_partner_id`?

Reply:
@alexey-pelykh Done !!